src/Database/Adapter/MariaDB.php

```diff
@@ -1181,7 +1183,11 @@ public function count(string $collection, array $queries = [], ?int $max = null,
 
         $stmt->execute();
 
-        $result = $stmt->fetch();
+        $result = $stmt->fetchall();
```

Comment:
Let's make sure we use proper casing:

```suggestion
        $result = $stmt->fetchAll();
```